docs-2.0/nebula-explorer/workflow/2.create-workflow.md

```diff
@@ -0,0 +1,56 @@
+  |Input| Set custom parameters that can be used for parameterized query. Click **Add parameter** to add more custom parameters. |
+  |Query language| Select the graph space to execute the nGQL statement and fill in the nGQL statement. Click **Parse Parameter** to display the returned column name in the **Output**.|
+  |Output| The column name returned by parsing the query language. You can change the name, which is equivalent to aliasing the column with `AS`.|
+  |Results| Set the saving project of the result. To call the results expediently for other algorithms, the results of the graph query component can only be saved in the HDFS.|
+
+  !!! note
+
+        The connection anchors are shown in yellow, indicating that it is optional and can be set by user or provided by any other component.
+
+4. In the component library list on the left side of the process canvas page, select **Node importance**->**PageRank** and drag it onto the canvas. Connect the anchor `output0` to the anchor `input0` and the anchor `output1` to the anchor `input1`.
+
+  ![connect-component](https://docs-cdn.nebula-graph.com.cn/figures/ex-connect-220623-cn.png)
+
+5. Click the graph computing component and set the following parameters in the configuration panel on the right side.
+
+  ![connect-component](https://docs-cdn.nebula-graph.com.cn/figures/ex-algorithm-220623-en.png)
+
+  |Parameters|Description|
+  |:---|:---|
+  |PageRank|Click ![pencil](https://docs-cdn.nebula-graph.com.cn/figures/workflow-edit.png) to modify the component name to identify the component.|
+  |Input| Three data sources are supported as input. <br>**Nebula Graph**: Users must select the graph space and corresponding edge types.<br>**Dependence**: The system will automatically recognize  the data source according to the connection of the anchor.<br>**HDFS**: Users must select HDFS and fill in the relative path of the data source file.|
+  |Parameter settings| Set the parameters of the graph algorithm. The parameters of different algorithms are different. Some parameters can be obtained from any upstream component where the anchor are shown in yellow.|
+  |Output| Display the column name of the graph computing results. The name can not be modified.|
+  |Execution settings| **Machine num**: The number of machines executing the algorithm.<br>**Processes**: The total number of processes executing the algorithm. Allocate these processes equally to each machine based on the number of machines.<br>**Threads**: How many threads are started per process.|
+  |Results| Set the result saved in HDFS or Nebula Graph.<br>**HDFS**: The save path is automatically generated based on the job and task ID.<br>**Nebula Graph**: Tags need to be created beforehand in the corresponding graph space to store the results. For more information about the properties of the tag, see [Algorithm overview](../../graph-computing/algorithm-description.md).<br>Some algorithms can only be saved in the HDFS.|
```

Comment:
Set the result saved in HDFS or Nebula Graph.

Set the restoration path of the results in HDFS or Nebula Graph?